.gitignore

```diff
@@ -1,2 +1,3 @@
 doc/documentation/python_api
 doc/documentation/cpp_api
+.DS_Store
```

Comment:
please add system specific files to be ignored in your global git config: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Done.